spec/factories/tasks.rb

```diff
@@ -6,4 +6,10 @@
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
   end
+
+  trait :task_finished_status do
```

Comment:
「ステータスが完了になっていること」以外にも「完了日時が登録されていること」などの要素があるので「finished」などのtrait名の方が分かりやすいかもしれません。

また今回は`done`というステータスがあるので、`finished`よりはステータス名の`done`に合わせた方が良いでしょう。
```suggestion
  trait :done do
```